### ISSUE TITLE
HA Warnings at startup - "Detected Blocking call to import_module..."

### ISSUE THREAD

**Issue by GaryOkie** [Vaskivskyi/ha-asusrouter#823]
### The problem

These ha-asusrouter warnings show up on startup with HA 2024.5.   Warning requests creating a bug report - so, here you go...  (thanks for reviewing)

```2024-05-03 17:44:29.443 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 149: await self.api.async_connect() (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.050 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.337 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.347 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 488: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.376 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.387 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.444 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
2024-05-03 17:44:30.459 WARNING (MainThread) [homeassistant.util.loop] Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
```

### Your device model

RT-AX86U

### Firmware type

Merlin

### Firmware version

3004.388.4

### Integration version with the issue

v 0.31

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

2024.5.1

### Is there anything useful in the logs?

```txt
if needed, will enable debug and resubmit logs upon request.
```

### Diagnostics information

Provided only upon request

**Comment by GaryOkie** [comment]
possibly related - https://github.com/frenck/spook/pull/722

**Comment by smarthomefamilyverrips** [comment]
Same here.

Logger: homeassistant.util.loop
Source: util/loop.py:99
First occurred: 13:22:24 (6 occurrences)
Last logged: 13:22:29

Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 149: await self.api.async_connect() (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 488: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues

**Comment by sezlony** [comment]
+1:

```
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 149: await self.api.async_connect() (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 488: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
```

**Comment by Gugulator** [comment]
have same issue in HA logs

`Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 149: await self.api.async_connect() (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 272: raw = await self.api.async_get_data(datatype, force=force) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 512: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues
Detected blocking call to import_module inside the event loop by custom integration 'asusrouter' at custom_components/asusrouter/bridge.py, line 488: data = await self.api.async_get_data(datatype) (offender: /usr/local/lib/python3.12/site-packages/asusrouter/modules/endpoint/__init__.py, line 61: submodule = importlib.import_module(module_name)), please create a bug report at https://github.com/Vaskivskyi/ha-asusrouter/issues`

**Comment by Vaskivskyi** [comment]
Hello, guys. Sorry for the late reply. Currently, I don't have any time to check GitHub regularly due to some personal reasons

The fix is ready and will be released either later today or tomorrow to remove the warning from HA logs

**Comment by Vaskivskyi** [comment]
The warning is fixed in `0.32.0` which is already available in HACS. Please, check it
